lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -871,7 +871,7 @@ fc_extras
 
     CM_PARSE = re.compile(  r'''
                                 (?P<name>([\w_]+\s*:\s+)+)
-                                (?P<method>[\w_\s]+(?![\w_]*:))\s*
+                                (?P<method>[\w_\s]+(?![\w_]*\s*:))\s*
```

Comment:
@pelson - I can't help being curious... what are you after from the non-greedy form? Performance?